includes/discovery/os/trapeze.inc.php

```diff
@@ -0,0 +1,14 @@
+<?php
+//
+// Discovery of Juniper Wireless (Trapeze) devices.
+//
+if (!$os) {
+    if (preg_match('/^Juniper Networks/', $sysDescr)) {
+        if (str_contains($sysDescr, 'MX-')) {
```

Comment:
Juniper also have MX routers but they are named "MX50" or "MX240" without "-".
I will have a look at it and see if there is other ways to determine if it's a Trapeze/Juniper Wireless device.